clientd3d/annotate.c

```diff
@@ -128,70 +128,60 @@ void MapMoveAnnotations( MapAnnotation *annotations, int x, int y, float scale,
  *   edit annotation that was clicked on, or add a new annotation.
  *		Modified by ajw 5/19/97. Annotations made possible only in MiniMap.
  */
-void MapAnnotationClick(int x, int y)
-{
+void MapAnnotationClick(int x, int y) {
   int i, index;
   MapAnnotation *a;
-  Bool existed;    // True if editing an existing annotation
-  
-//  int stretchfactor = config.large_area ? 2 : 1;
+  Bool existed; // True if editing an existing annotation
```

Comment:
Removing tabs is fine but please leave at least 2 spaces before trailing comments, otherwise it's hard to see the end of the code.